site/content/en/docs/tasks/manage/restricting_job_creation.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Restricting job creation with Validating Admission Policy"
+date: 2024-06-28
+weight: 10
+description: >
+  Implementing Validating Admission Policy to prevent job creation without queue name.
+---
+
+This page shows you how to set up a [Validating Admission Policy](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy) to prevent the creation of jobs without a queue name within a namespace.
+
+## Before you begin
+
+Ensure the following conditions are met:
+
+- A Kubernetes cluster is running.
+- The kubectl command-line tool can communicate with your cluster.
+
+## Creating a Validating Admission Policy
+
+To verify a job contains a queue name, check that the `kueue.x-k8s.io/queue-name` label exists and has a value set using the [Common Expression Language (CEL)](https://github.com/google/cel-spec).
+
+Here is an example of a ValidatingAdmissionPolicy that rejects job creation without a queue name:
+
+```yaml
+# sample-validating-policy.yaml
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: sample-validating-admission-policy
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   ["batch"]
+      apiVersions: ["v1"]
+      operations:  ["CREATE", "UPDATE"]
+      resources:   ["jobs"]
+  validations:
+    - expression: "object.metadata.labels.exists_one(key, key == 'kueue.x-k8s.io/queue-name') && object.metadata.labels['kueue.x-k8s.io/queue-name'] != ''"
```

Comment:
```suggestion
    - expression: "'kueue.x-k8s.io/queue-name' in object.metadata.labels && object.metadata.labels['kueue.x-k8s.io/queue-name'] != ''"
```
Shorter, and looks equivalent given that an object cannot have more than one value for a key.